source/grammar/qasm3.g4

```diff
@@ -237,8 +237,8 @@ quantumMeasurement
     ;
 
 quantumMeasurementAssignment
-    : quantumMeasurement ( ARROW indexIdentifierList)?
-    | indexIdentifierList EQUALS quantumMeasurement
+    : quantumMeasurement ARROW indexIdentifier
```

Comment:
```suggestion
    : quantumMeasurement ( ARROW indexIdentifier )?
```
Need this to new optional. Something like `measure a`, without assignment, is valid.